dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/JSONFieldsTest.java

```diff
@@ -118,7 +118,7 @@ public void testMessageFieldNamesOmitEnv() throws Exception {
         setUp(server_env_omit);
         //List<String> lines = server_env_omit.findStringsInFileInLibertyServerRoot("\"message\"", MESSAGE_LOG);
         //assertTrue("The message field name was not omitted in messages.log.", lines.size() == 0);
-        assertNull("The message field name was not omitted in messages.log.", server_xml.waitForStringInLogUsingMark("\"loglevel\""));
+        assertNotNull("The loglevel field name was not omitted in messages.log.", server_env_omit.waitForStringInLogUsingMark("^.*\"module\":((?!\"loglevel\").)*?"));
```

Comment:
the regex we tested was `^.*module:((?!ibm_sequence).)*$` -- I understand why you have added the quotes around `module` and put `loglevel` in instead of `ibm_sequence`, but I think it should be `$` at the end rather than `?`